### DISCUSSION TITLE
Support for snippets that can be used in many pages

### DISCUSSION THREAD

**Discussion by mcgaritydotme** [GitbookIO/community#121] [Feature Requests]
### What's your feature/idea?

**Snippets** (e.g. excerpts) would be pieces of content which are defined once, but can be inserted into any number of other articles. When those articles are published, the snippet content is included seamlessly. And if the snippet itself is updated, any articles referencing the snippet automatically pick up its updates.

Today, we have several articles which repeat the same content. Examples include: shared steps for a particular screen &  introductory paragraphs.  If I were to use GitBook, I'd have to manually repeat this content. And if I ever need to update that shared content, I must touch several articles (assuming I even remember which articles need these updates).

### What's your desired outcome?

Here's an example of how snippets work within Notion, just to illustrate the effect of updating one location & having others reflect it:

<img width="1512" alt="Screenshot" src="https://user-images.githubusercontent.com/450236/221567181-380f0716-b792-411b-8edb-8b973c567cbf.png">

In my case, I'd have several articles (like on the right) that would leverage the same shared content (left). 

### What's the impact of this for you?

- [X] Critical
- [ ] High
- [ ] Medium
- [ ] Low

### Anything else?

This is the one feature that prevents us from adopting GitBook.

**Comment by igalloway** [comment]
I would like to upvote this issue. We have several user guides where certain procedures are repeated. An example of this is step by step details on how to connect a debugger tool to a circuit board and get the software working, but then the actual details of WHAT to program need to be individualized to the specific board. 
Sometimes a detail or additional clarification is needed in the debugger details, and we really dislike repeating and copying the same instructions to half a dozen different gitbooks. We have considered breaking out common tasks to a separate gitbook, but that feels cumbersome and also it creates a "frankentein" gitbook with a bunch of potentially unrelated procedures in it. 

Being able to insert snippets that could be included in muliple gitbooks would be very much appreciated. It is our most bothersome issue at the moment.

**Comment by jenningsb2** [comment]
Writing in to request this feature as well! Support for transclusions would be a game changer. 

Our API docs reference the steps to obtain and API and authenticate across multiple pages in our docs. 

At the moment I have opted to just linking directly to the header of the main authentication doc to save myself the maintenance headache of have to update the same section in 5 different pages.

**Comment by MadeByPP** [comment]
surely. Without references, it would be difficult to keep your documentation consistent. This is an essential function for document-oriented services.

**Comment by thebiglabasky** [comment]
Hi everyone, a short update to let you know we're actively working on this and we should land something in the next few weeks. It'll be called "Synced blocks"

**Reply by sianchai** [reply]
hi @thebiglabasky, wondering if there has been any update on this feature? thanks!

**Reply by ParsecSupport** [reply]
Hi @thebiglabasky,

We are also wondering if there is a date when Synced blocks will be available. Appreciate any info you can provide.

**Reply by thebiglabasky** [reply]
I’ll let @SamyPesse take the lead on this one!

**Reply by ParsecSupport** [reply]
Can we please get an answer?

**Reply by taszap** [reply]
Hello @ParsecSupport Thanks for reaching out! 
I don't think we can share a specific date just yet. Synced blocks are on our roadmap, and are been worked on but we have not set a date for the release. We will share the announcement as soon as we are ready!

**Reply by zdmilot** [reply]
Very exciting and the feature is a fantastic addition to gitbook! thank you @taszap and @thebiglabasky for working on this feature, can't wait for this to roll out fully!

**Comment by SamyPesse** [comment]
We just released **Reusable content**: https://github.com/orgs/GitbookIO/discussions/772 🎉 

Let us know what you think in the announcement post.
